tools/cli/flags.go

```diff
@@ -354,6 +356,15 @@ func getFlagsForStart() []cli.Flag {
 				"If value is array, use json array like [\"a\",\"b\"], [1,2], [\"true\",\"false\"], [\"2019-06-07T17:16:34-08:00\",\"2019-06-07T18:16:34-08:00\"]. " +
 				"Use 'cluster get-search-attr' cmd to list legal keys and value types",
 		},
+		cli.IntFlag{
+			Name:  FlagRetryAttempts,
+			Usage: "Optional retry attempts. If set workflow will be retried the specified amount of times.",
```

Comment:
maybe add to usage: "...  with fixed retry interval"

Reply:
Either this or additional backoff option. For now added backoff option as suggested by @yycptt.